psi4/src/psi4/libmints/extern.cc

```diff
@@ -92,6 +92,25 @@ void ExternalPotential::print(std::string out) const {
     }
 }
 
+std::string ExternalPotential::get_xyz() {
+
+    std::string xyz = "";
+    size_t n_atoms = 0;
+    // Charges
+    if (charges_.size()) {
+        for (size_t i = 0; i < charges_.size(); i++) {
```

Comment:
This looks like a good time to use a range-based for loop.

Reply:
Fixed.